deepcell/utils/io_utils.py

```diff
@@ -149,8 +149,13 @@ def get_images_from_directory(data_location, channel_names):
     all_images = []
 
     for stack_iteration in range(len(img_list_channels[0])):
-        if data_format == 'channels_first':
+
+        if (data_format == 'channels_first') and (img_temp.ndim > 2):
+            shape = (1, n_channels, img_temp.shape[0], img_temp.shape[1], img_temp.shape[2])
+        elif data_format == 'channels_first':
             shape = (1, n_channels, img_temp.shape[0], img_temp.shape[1])
+        elif img_temp.ndim > 2:
+            shape = (1, img_temp.shape[0], img_temp.shape[1], img_temp.shape[2], n_channels)
```

Comment:
Is this just for loading 3d data? what was the issue here?

Reply:
Yeah it's for loading 3D data, would break if the image didn't have ndim==2

Reply:
I think we can clean up the `if` blocks by not enumerating each shape but just concatenating lists.

```python
if data_format == 'channels_first':
    shape = tuple([1, n_channels] + list(img_temp.shape))
else:
    shape = tuple([1] + list(img_temp.shape) + [n_channels])
```

Reply:
Done